docs/graphql/manual/guides/deployment/render.rst

```diff
@@ -0,0 +1,115 @@
+Hasura GraphQL Engine on Render
+====================================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+This guide talks about how to deploy the Hasura GraphQL engine on `Render
+<https://render.com>`__.
+
+
+Pre-requisites
+--------------
+
+- GitHub (or alternative) account
+- Render account associated with a Github account and with billing enabled
+
+
+Create a repository
+-------------------
+
+Create a new GitHub repository
```

Comment:
Can we have bullet points on the instructions here.

Reply:
Attempted fix, but I'm worried I didn't do this right.